scorched/connection.py

```diff
@@ -286,9 +282,7 @@ def _prepare_docs(self, docs):
                 # fields
                 if value is None:
                     continue
-                if name in self._datefields:
```

Comment:
This should be still in. If a `name` is in `self._datefields` we already know its type `date`